com.matthiasn.lotti.yml

```diff
@@ -0,0 +1,240 @@
+  - -Dgtk_doc=false
+  - -Dintrospection=false
+  sources:
+  - type: archive
+    sha256: b33b9542222ea8866f6ff2d31c0ad373877c2277db546ca00cc7fdda9cbab1c3
+    path: libsecret-0.20.5.tar.gz
+- name: libkeybinder
+  buildsystem: autotools
+  sources:
+  - type: archive
+    sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+    path: keybinder-3.0-0.3.2.tar.gz
+- name: noto-emoji-fonts
+  buildsystem: simple
+  build-commands:
+  - mkdir -p /app/share/fonts/truetype
+  - mkdir -p /app/etc/fonts/conf.d
+  - install -Dm644 NotoColorEmoji.ttf /app/share/fonts/truetype/NotoColorEmoji.ttf
+  - install -Dm644 75-noto-color-emoji.conf /app/etc/fonts/conf.d/75-noto-color-emoji.conf
+  - fc-cache -fv /app/share/fonts
+  sources:
+  - type: file
+    sha256: 72a635cb3d2f3524c51620cdde406b217204e8a6a06c6a096ff8ed4b5fd6e27b
+    path: NotoColorEmoji.ttf
+  - type: file
+    path: 75-noto-color-emoji.conf
+- rustup-1.83.0.json
+- name: lotti
+  buildsystem: simple
+  build-options:
+    append-path: /var/lib/rustup/bin:/var/lib/flutter/bin:/usr/lib/sdk/rust-stable/bin:/run/build/lotti/.cargo/bin:/usr/lib/sdk/llvm20/bin
+    prepend-ld-library-path: /usr/lib/sdk/llvm20/lib
+    env:
+      PUB_CACHE: /run/build/lotti/.pub-cache
+      FLUTTER_ROOT: /run/build/lotti/flutter_sdk
+      PATH: /var/lib/rustup/bin:/usr/lib/sdk/rust-stable/bin:/var/lib/flutter/bin:/run/build/lotti/flutter_sdk/bin:/run/build/lotti/.cargo/bin:/usr/lib/sdk/llvm20/bin:/usr/bin:/bin
+      CARGO_HOME: /run/build/lotti/.cargo
+      RUSTUP_HOME: /var/lib/rustup
+      PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/lib64/pkgconfig:/usr/lib/pkgconfig:/usr/lib64/pkgconfig
+  build-commands:
+  - echo "Setting up Flutter SDK..."
+  - if [ -d /var/lib/flutter ]; then cp -r /var/lib/flutter /run/build/lotti/flutter_sdk;
+    fi
+  - chmod -R u+w /run/build/lotti/flutter_sdk
+  - echo "Installing Rust..."
+  - rustc --version
+  - cargo --version
+  - echo "Building Lotti from source..."
+  - mkdir -p .cargo
+  - ln -sfn ../cargo .cargo/cargo
+  - cp cargo/config .cargo/config.toml 2>/dev/null || true
+  - /run/build/lotti/flutter_sdk/bin/flutter pub get --offline
+  - echo "Checking for libsecret installation..."
+  - pkg-config --modversion libsecret-1 || echo "libsecret-1 not found via pkg-config"
+  - ls -la /app/lib/pkgconfig/ | grep secret || echo "No libsecret pkgconfig files
+    in /app/lib/pkgconfig/"
+  - export CMAKE_PREFIX_PATH=/app:$CMAKE_PREFIX_PATH
+  - /run/build/lotti/flutter_sdk/bin/flutter build linux --no-pub --release --verbose
+  - echo "Installing Flutter bundle..."
+  - "arch=\"x64\"\nif [ \"${FLATPAK_ARCH}\" = \"aarch64\" ]; then\n  arch=\"arm64\"\
+    \nfi\ncp -r build/linux/${arch}/release/bundle/* /app/\n"
+  - find /app -name "*1024*" -type f -delete
+  - chmod +x /app/lotti
+  - mkdir -p /app/bin
+  - mkdir -p /app/lib/ffmpeg
+  - "cat > /app/bin/lotti << 'EOF'\n#!/bin/bash\nexport LD_LIBRARY_PATH=/app/lib:/app/lib64:$LD_LIBRARY_PATH\n\
```

Comment:
Never use shell commands to generate a script. Use `type: script` or directly a file.